packages/cli/src/util/create-git-meta.ts

```diff
@@ -93,6 +93,11 @@ export function isDirty(directory: string, output: Output): Promise<boolean> {
 
 export async function parseGitConfig(configPath: string, output: Output) {
   try {
+    const configExists = await fs.pathExists(configPath);
+    if (!configExists) {
+      return;
+    }
```

Comment:
This prevents the unhelpful error message (from line 103) from showing up when there's no `git.config` to even try to parse.